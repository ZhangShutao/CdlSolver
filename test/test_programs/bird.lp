fly(X) :- bird(X), &c{fly(X)}.
-fly(X) :- penguin(X).
:- fly(X), ostrich(X).
bird(tweety). penguin(tweety).
bird(plucky). ostrich(plucky).
bird(ringo).

#show fly/1.