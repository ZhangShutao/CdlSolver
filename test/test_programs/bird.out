stable model: 1
X:bird(tweety), fly(tweety);
A:fly(tweety);
Satisfiable